low-level/framework/utils/ipmi_client.py

```diff
@@ -288,14 +288,11 @@ def is_fru(self, fru):
         if is_fru:
             if fru in self.hot_swapped_frus:
                 fru_str = str(is_fru).lower() + ":" + "hot_swappable"
-            elif fru in self.cold_swapped_frus:
-                fru_str = str(is_fru).lower() + ":" + "cold_swappable"
             else:
-                fru_str = str(is_fru).lower() + ":" + "unknown"
+                fru_str = str(is_fru).lower() + ":" + "cold_swappable"
```

Comment:
addressing the same change for Server FRU representation as well in this PR only